pkg/backend/updates.go

```diff
@@ -77,6 +77,12 @@ const (
 
 	// CISystem is the name of the CI system running the pulumi operation.
 	CISystem = "ci.system"
+	// CIBuildID is an opaque ID of the build in the CI system.
+	CIBuildID = "ci.build.id"
+	// CIBuildType is the type of build of the CI system, e.g. "push", "pull_request", "test_only".
```

Comment:
Note that the Travis event types are `push`, `pull_request`, `cron` and `api`.  I'm not sure what `test_only` is here

Reply:
Yes, this is expected to be a CI-system specific value. And `test_only` would be some hypothetical value some future system would have.